workflow/rules/annotation.smk

```diff
@@ -22,6 +22,9 @@ rule annotate_variants:
         calls="results/calls/{group}.{scatteritem}.bcf",
         cache="resources/vep/cache",
         plugins="resources/vep/plugins",
+        REVEL=lambda wc: get_plugin_aux("REVEL"),
+        revel_tbi=lambda wc: get_plugin_aux("REVEL", True),
+        LoFtool=lambda wc: get_plugin_aux("Loftool"),
```

Comment:
These are case sensitive as the VEP snakemake-wrapper is looking for plugin aux-files in the input by looking for the plugin-name.
Alternatively, we could set this to lowercase in the wrapper but this might lead to confusion for users.
Any suggestions?